requirements/doc-requirements.txt

```diff
@@ -0,0 +1,78 @@
+# NOTE: These requirements are used for developing code on the repo.
```

Comment:
Are those essentially `requirements` + `sphinx`?

Reply:
Yes, these are the dev-requirements plus sphinx and minus the weird local path dependent package installation.

Would there be a cleaner way to do this? Maybe we should just remove the weird path dependency in dev-requirements (by changing dir in make) and add sphinx to dev? In development we might also want to be able to build docs locally?

Reply:
Not quite sure bc. I'm not as familiar with Sphinx - are those run anywhere? 
I think in any case there are several `dev-requirements` that are not needed anymore. I did not update the `dev-requirements` for now, but the `requirements` is the minimal list of requirements that are needed to have GeoGraph run

Reply:
I guess sphinx probably needs even fewer as it doesn't even execute the geograph code?

Reply:
Cool! Let's return to this when we define all the requirements for the package itself. We can then do this in a separate PR.

Reply:
Sounds good to me!